content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md

```diff
@@ -3,70 +3,100 @@ reviewers:
 - bsalamat
 - k82cn
 - ahg-g
-title: Resource Bin Packing for Extended Resources
+title: Resource Bin Packing
 content_type: concept
 weight: 80
 ---
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
 
-The kube-scheduler can be configured to enable bin packing of resources along
-with extended resources using `RequestedToCapacityRatioResourceAllocation`
-priority function. Priority functions can be used to fine-tune the
-kube-scheduler as per custom needs. 
+In the [scheduling-plugin](/docs/reference/scheduling/config/#scheduling-plugins)  `NodeResourcesFit` of kube-scheduler, there are two 
+scoring strategies that support the bin packing of resources: `MostAllocated` and `RequestedToCapacityRatio`.
 
 <!-- body -->
 
-## Enabling Bin Packing using RequestedToCapacityRatioResourceAllocation
+## Enabling Bin Packing using MostAllocated strategy
```

Comment:
We should follow the style guide recommendation to use sentence casing for headers. Check out [Style guide section on headers](https://kubernetes.io/docs/contribute/style/style-guide/#headings). Revised header should be "Enabling bin packing using MostAllocated strategy". Same for header further down.

Reply:
Done